crates/swc_ecma_parser/src/parser/expr/ops.rs

```diff
@@ -294,19 +294,28 @@ impl<I: Tokens> Parser<I> {
             }
 
             if self.input.syntax().typescript() && op == op!("delete") {
-                fn unwrap_paren(e: &Expr) -> &Expr {
-                    match *e {
-                        Expr::Paren(ref p) => unwrap_paren(&p.expr),
-                        _ => e,
+                fn unwrap_paren_iterate(e: &Expr) -> &Expr {
```

Comment:
I think this is identical as https://rustdoc.swc.rs/swc_ecma_ast/enum.Expr.html#method.unwrap_parens

Reply:
Maybe we should change it to iterate, instead of recursing.